pydev_ipython/inputhook.py

```diff
@@ -26,6 +26,7 @@
 GUI_QT = "qt"
 GUI_QT4 = "qt4"
 GUI_QT5 = "qt5"
+GUI_QT6 = "qt6"
```

Comment:
All the changes in this directory were changes made in Debugpy before I started this work, so this directory can be ignored.